vsts/scripts/pullAndTag.sh

```diff
@@ -9,23 +9,44 @@ set -euo pipefail
 # $2 > runtime-images-acr.txt
 declare imagefilter="oryxdevmcr.azurecr.io/public/oryx"
 
-echo "Build image filter is set"
-while read buildImage; do
-  # Always use specific build number based tag and then use the same tag to create a 'latest' tag and push it
-  if [[ $buildImage != *:latest ]]; then
-	echo "Pulling the build image $buildImage ..."
-	docker pull "$buildImage" | sed 's/^/     /'
+function tagBuildImageForIntegrationTest() {
+	local devbuildImageName="$1"
+	local devbuildImageType="$2"
+	local buildDefName="$BUILD_DEFINITIONNAME"
+	local buildNumber="$BUILD_BUILDNUMBER"
```

Comment:
@kichalla ... does this buildnumber need to be $RELEASE_TAG_NAME? I was not 100% sure but I felt like it should be because of patch builds etc??

Reply:
Yes we want to use `$RELEASE_TAG_NAME` for tagging the images